packages/nx/src/plugins/js/utils/register.ts

```diff
@@ -44,6 +46,10 @@ export function getSwcTranspiler(
   const register = require('@swc-node/register/register')
     .register as ISwcRegister;
 
+  if (existsSync(join(workspaceRoot, 'tsconfig.base.json'))) {
+    process.env.SWC_NODE_PROJECT = 'tsconfig.base.json';
```

Comment:
Shouldn't this be the full path?

Reply:
No, it works without because swc does a `resolve(tsconfigPath)` https://github.com/swc-project/swc-node/blob/9f674cd67091192b1fe62befd13cf4b61a6377e5/packages/register/read-default-tsconfig.ts#L26